conversions/views.py

```diff
@@ -0,0 +1,83 @@
+import csv
+import os
+from io import StringIO
+
+from django.http import HttpResponse
+from django.shortcuts import render
+
+from codelists.coding_systems import CODING_SYSTEMS
+from mappings.ctv3sctmap2.mappers import get_mappings
+
+from .forms import ConvertForm
+
+
+def convert(request):
+    if request.POST:
+        form = ConvertForm(request.POST, request.FILES)
+        if form.is_valid():
```

Comment:
There's no `else` to this check so the invalid form isn't handled, as below that would probably be better served as a shortcut return.